Solutions/CohesitySecurity/Tests/az.test.py

```diff
@@ -1,16 +1,65 @@
 #!/usr/bin/env python3
 
+import subprocess
 import json
+import time
 from az import *
+from helios import *
 import os
 
 os.chdir(os.path.dirname(os.path.abspath(__file__)))
 f = open('../cohesity.json',)
 data = json.load(f)
 resource_group = data['resource_group']
 workspace_name = data['workspace_name']
-playbook_name = data['playbook_name']
 apiKey = data['apiKey']
 f.close()
 
 assert has_dup_incidents(resource_group, workspace_name) == False
+
+
+def test_cohesity_close_helios_incident():
+    playbook_name = "Cohesity_Close_Helios_Incident"
+    subscriptionId = get_subscriptionId()
+    vid, alert_id = get_one_incident_id(resource_group, workspace_name)
+
+    alert_details = get_alert_details(alert_id, apiKey)
+    assert alert_details['alertState'] == "kOpen"  # maybe we need to close incident after close helios alert, otherwise, this assert might fail.
+    returncode = run_playbook(subscriptionId, vid, resource_group, workspace_name, playbook_name)
+    assert returncode == 0
+
+    time.sleep(30)  # Sleep for 30 seconds
+
+    alert_details = get_alert_details(alert_id, apiKey)
+    print("alert_id --> %s" % alert_id)
+    print("apiKey --> %s" % apiKey)
```

Comment:
Do we really need to publish the API key (data that can be used for intrusion)? If we do, then maybe you can limit it to first 4 and last 4 symbols?

Reply:
this is just local printing, which should be fine.
the intruder could still get it from local cohesity.json file.
we need this info to debug, when test case failed.